packages/neo-one-website/src/elements/Markdown.tsx

```diff
@@ -1,5 +1,7 @@
 import MarkdownIt from 'markdown-it';
-import anchor from 'markdown-it-anchor';
+// @ts-ignore
```

Comment:
this is a workaround for a similar issue this lib is having. https://github.com/nuxt/vue-meta/issues/25 supposedly it was fixed https://github.com/valeriangalliat/markdown-it-anchor/issues/12 but looks like its back.